source/clear-linux/get-started/bootable-usb/bootable-usb-linux.rst

```diff
@@ -13,56 +13,97 @@ Alternative instructions for other operating systems are available:
-   the image in order to verify its integrity.
+.. _image: https://download.clearlinux.org/image
 
-.. _verify-checksum:
+.. include ../../guides/maintenance/download-image.rst
+..   :Start-after: download-usb-suitable-images:
+..   :end-before: verify-image-checksum
 
-Verify the integrity of the download (recommended)
-==================================================
+.. Download the latest Clear Linux image
+.. =====================================
 
-#. Start a terminal emulator.
-#. Go to the directory with the downloaded files.
-#. To verify the integrity of the image, enter the following (a live image
-   is used as example):
+.. There are 2 types of |CL| images that are suitable for burning onto and running 
+.. off a USB drive:
 
-   .. code-block:: console
+..   * Live image: `clear-<version number>-live.img.xz`
+..   * Installer image: `clear-<version number>-installer.img.xz`
+
+.. Go to the Clear Linux `image`_ repository and download the desired type.
+
+..   * Live image: `clear-<version>-live.img.xz`
+..   * Installer image: `clear-<version>-installer.img.xz`
+
+..   For older versions, see the `releases`_ page.
+
+.. Although not required, it is recommended to download the corresponding
+..   checksum file (designated with `-SHA512SUMS` at the end of the filename) for
+..   the image in order to verify its integrity.
+
+.. verify-checksum
 
-      $ sha512sum ./clear-<version>-live.img.xz | diff ./clear-<version>-live.img.xz-SHA512SUMS -
+.. _end-download-usb-suitable-images:
 
-   If the checksum of the downloaded image is different than the original
-   checksum, the differences will displayed. An empty output indicates a match.
+.. include:: ../../guides/maintenance/download-image.rst
+   :Start-after: verify-image-checksum:
+   :end-before: verify-image-checksum-on-macos
+
+.. include:: ../../guides/maintenance/download-image.rst
+   :Start-after: uncompress-image:
+   :end-before: uncompress-gz-on-linux
+
+.. Verify the integrity of the download (recommended)
+.. ==================================================
+
+.. #. Start a terminal emulator.
+.. #. Go to the directory with the downloaded files.
+.. #. To verify the integrity of the image, enter the following (a live image
+..   is used as example):
+
+.. code-block console
+
+..      $ sha512sum ./clear-<version>-live.img.xz | diff ./clear-<version>-live.img.xz-SHA512SUMS -
+
+..   If the checksum of the downloaded image is different than the original
+..   checksum, the differences will displayed. An empty output indicates a match.
 
 .. _copy-usb-linux:
 
 Burn the Clear Linux image onto a USB drive
 ===========================================
 
-.. caution::
+.. _caution-burn-usb-drive:
 
-   Backup important data before proceeding. The process of burning an image
-   onto the USB drive completely formats the
-   USB drive and any existing content will be destroyed.
+**Caution:** Backup important data before proceeding. The process of burning 
```

Comment:
The caution directive must be employed! It is internationally standardize that caution notices must have a specific look and must be written following this template:
Signal word: Caution! [This is included automatically by the directive]
What is the danger?: Burning the image formats the USB drive destroying all existing content.
How to avoid? : Backup your data before proceeding.